### PR TITLE
libreoffice-qt: init at 6.4.3.2

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by peterhoeg** [comment]
So the PR description was hosed - sorry about the noise.

`libreoffice-qt` looks *much* nicer than it's GTK3 cousin when run under KDE.

Fixes #57846

Cc: @7c6f434c @ajs124 @mmilata

**Comment by peterhoeg** [comment]
Relates to #73387 and #68414

**Comment by peterhoeg** [comment]
Only building the `fresh` flavour but there is nothing preventing us from doing `still` as well.

**Review by ajs124 — APPROVED** [review]
We might want to do fresh and still, but I'm fine with only fresh.

Compiled and ran. This does indeed look nicer on KDE.

File: pkgs/applications/office/libreoffice/default.nix [hunk 1]
```diff
@@ -57,8 +60,9 @@ in (stdenv.mkDerivation rec {
 
   # For some reason librdf_redland sometimes refers to rasqal.h instead
   # of rasqal/rasqal.h
-  NIX_CFLAGS_COMPILE = "-I${librdf_rasqal}/include/rasqal"
-    + stdenv.lib.optionalString stdenv.isx86_64 " -mno-fma";
+  NIX_CFLAGS_COMPILE = [
```
Comment (jtojnar) [inline pkgs/applications/office/libreoffice/default.nix:63]: Environment variables need to be strings, this will be an error with structured attrs.

Comment (peterhoeg) [reply pkgs/applications/office/libreoffice/default.nix:63]: I had a read through https://nixos.mayflower.consulting/blog/2020/01/20/structured-attrs/ which was really quite interesting. Don't we consider a list a primitive type on nix?

Comment (jtojnar) [reply pkgs/applications/office/libreoffice/default.nix:63]: Reasoning here: https://github.com/NixOS/nixpkgs/pull/72074#issuecomment-609435309

Comment (peterhoeg) [reply pkgs/applications/office/libreoffice/default.nix:63]: Thanks for the pointers, I'll open a separate issue to address this.

**Pull request merged**